### ISSUE TITLE
DataArray to_dfs() method crashes when its name contains '*'

### ISSUE THREAD

**Issue by rywm-dhi** [DHI/mikeio#416]
**Describe the bug**
As the title says. For example, a DataArray with the name "Total water depth * Current speed" will crash a jupyter kernel when it calls to_dfs(). This didn't happen in v1.03, but happens when installing the current main branch.

**To Reproduce**
Open a dfsu file, multiply two items, and take their nanmax. The default DataArray name will be the name of the two items separated by a '*'. Use to_dfs() on the DataArray and the error should show up.

If you change the DataArray to something without a '*' in it, then to_dfs() works.

Include code snippet
```
import mikeio
ds = mikeio.read(r"data\BlueBeach_100Year\100y_flowBase2D dynamics.dfsu", items=["Total water depth", "Current speed"])
da = (ds[0] * ds[1]).nanmax()
da.to_dfs("test.dfsu")
```

**Expected behavior**
For it to write the dfsu file as normal.

**Screenshots**
![image](https://user-images.githubusercontent.com/74630349/188659526-67f0bb8b-d082-4138-924d-6621bb1b0d55.png)

**System information:**
 - Python version 3.10.6
 - MIKE version v1.04 (installed from main branch)

**Comment by jsmariegaard** [comment]
Thanks for reporting @rywm-dhi ! I wonder if this is related to the wildcard selection of items which was added recently #383

**Comment by ecomodeller** [comment]
It is actually not the .to_dfs() that is the problem.

It is related to the wildcard selection: https://github.com/DHI/mikeio/blob/ca495a7692f982c9696f76b6d478ecac3adf67ac/mikeio/dataset.py#L711-L720

So what to do?
1. Forbid * in item names
2. Make wildcard selection more explicit
3. Fix it some other way

**Comment by ecomodeller** [comment]
Take a look at PR #417 where asterisks are allowed in item names but makes wildcard item selection using like this `ds["*foo*"]` impossible 🤔

**Comment by jsmariegaard** [comment]
😢 I loved the wildcard selection - I now miss it in pandas

**Comment by ecomodeller** [comment]
Fixed by #417
